operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -1911,6 +1911,321 @@ gateway:
         hosts:
           - gateway.gem.example.com
 
+graphite:
+  # If true, enables graphite querier and graphite write proxy functionality.
+  # Read more in https://grafana.com/docs/metrics-enterprise/latest/graphite/
+  enabled: true
+
+graphite_querier:
+  replicas: 2
+
+  schemasConfiguration:
+    storageSchemas: |-
+      [default]
+      pattern = .*
+      intervals = 0:1s
+      retentions = 10s:8d,10min:1y
+    storageAggregations: |-
+      [default]
+      aggregationMethod = avg
+      pattern = .*
+      xFilesFactor = 0.1
+
+  service:
+    annotations: {}
+    labels: {}
+
+  resources:
+    requests:
+      cpu: 100m
+      memory: 128Mi
+
+  # Additional Grafana Enterprise Metrics container arguments, e.g. log level (debug, info, warn, error)
+  extraArgs: {}
+
+  # Pod Labels
+  podLabels: {}
+
+  # Pod Annotations
+  podAnnotations: {}
+
+  nodeSelector: {}
+  affinity:
+    podAntiAffinity:
+      preferredDuringSchedulingIgnoredDuringExecution:
+        - weight: 100
+          podAffinityTerm:
+            labelSelector:
+              matchExpressions:
+                - key: target
+                  operator: In
+                  values:
+                    - graphite-querier
+            topologyKey: 'kubernetes.io/hostname'
+
+  annotations: {}
+  persistence:
+    subPath:
+
+  readinessProbe:
+    httpGet:
+      path: /ready
+      port: http-metrics
+    initialDelaySeconds: 45
+
+  securityContext: {}
+
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 0
+      maxUnavailable: 1
+
+  terminationGracePeriodSeconds: 180
+
+  env: []
+  extraEnvFrom: []
+  tolerations: []
+  podDisruptionBudget: {}
+  initContainers: []
+  extraContainers: []
+  extraVolumes: []
+  extraVolumeMounts: []
+  extraPorts: []
+
+graphite_write_proxy:
+  replicas: 2
+
+  service:
+    annotations: {}
+    labels: {}
+
+  resources:
+    requests:
+      cpu: 100m
+      memory: 128Mi
+
+  # Additional Grafana Enterprise Metrics container arguments, e.g. log level (debug, info, warn, error)
+  extraArgs: {}
+
+  # Pod Labels
+  podLabels: {}
+
+  # Pod Annotations
+  podAnnotations: {}
+
+  nodeSelector: {}
+  affinity:
+    podAntiAffinity:
+      preferredDuringSchedulingIgnoredDuringExecution:
+        - weight: 100
+          podAffinityTerm:
+            labelSelector:
+              matchExpressions:
+                - key: target
+                  operator: In
+                  values:
+                    - graphite-write-proxy
+            topologyKey: 'kubernetes.io/hostname'
+
+  annotations: {}
+  persistence:
+    subPath:
```

Comment:
Similar question as for the querier - does it need persistence? I don't see the `/data` dir being used anywhere in the config.

Reply:
They do, otherwise the pods fail to start. Its requirement of GEM.
```
level=info ts=2022-08-19T11:19:55.923295616Z caller=main.go:231 msg="starting Grafana Enterprise Metrics" version="(version=r197-c28774dd, branch=weekly-r197, revision=c28774dd)"
level=error ts=2022-08-19T11:19:55.92344861Z caller=server_util.go:26 original_caller=main.go:235 msg="error running Grafana Enterprise Metrics" err="open /data/metrics-activity.log: read-only file system\nfailed to create activity file\ngithub.com/grafana/mimir/pkg/util/activitytracker.getMappedFile\n\t/backend-enterprise/vendor/github.com/grafana/mimir/pkg/util/activitytracker/activity_tracker.go:177\ngithub.com/grafana/mimir/pkg/util/activitytracker.NewActivityTracker\n\t/backend-enterprise/vendor/github.com/grafana/mimir/pkg/util/activitytracker/activity_tracker.go:60\ngithub.com/grafana/mimir/pkg/mimir.(*Mimir).initActivityTracker\n\t/backend-enterprise/vendor/github.com/grafana/mimir/pkg/mimir/modules.go:141\ngithub.com/grafana/dskit/modules.(*Manager).initModule\n\t/backend-enterprise/vendor/github.com/grafana/dskit/modules/modules.go:120\ngithub.com/grafana/dskit/modules.(*Manager).InitModuleServices\n\t/backend-enterprise/vendor/github.com/grafana/dskit/modules/modules.go:92\ngithub.com/grafana/mimir/pkg/mimir.(*Mimir).Run\n\t/backend-enterprise/vendor/github.com/grafana/mimir/pkg/mimir/mimir.go:551\ngithub.com/grafana/backend-enterprise/pkg/enterprise/mimir/init.(*MimirEnterprise).Run\n\t/backend-enterprise/pkg/enterprise/mimir/init/mimir.go:158\nmain.main\n\t/backend-enterprise/cmd/enterprise-metrics/main.go:232\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:250\nruntime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1571\nerror initialising module: activity-tracker\ngithub.com/grafana/dskit/modules.(*Manager).initModule\n\t/backend-enterprise/vendor/github.com/grafana/dskit/modules/modules.go:122\ngithub.com/grafana/dskit/modules.(*Manager).InitModuleServices\n\t/backend-enterprise/vendor/github.com/grafana/dskit/modules/modules.go:92\ngithub.com/grafana/mimir/pkg/mimir.(*Mimir).Run\n\t/backend-enterprise/vendor/github.com/grafana/mimir/pkg/mimir/mimir.go:551\ngithub.com/grafana/backend-enterprise/pkg/enterprise/mimir/init.(*MimirEnterprise).Run\n\t/backend-enterprise/pkg/enterprise/mimir/init/mimir.go:158\nmain.main\n\t/backend-enterprise/cmd/enterprise-metrics/main.go:232\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:250\nruntime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1571"
```

Reply:
ok nevermind, I see it's using an empty dir anyways. you might as well keep this.